src/org/vitrivr/cineast/core/data/entities/MultimediaMetadataDescriptor.java

```diff
@@ -103,7 +104,14 @@ public MultimediaMetadataDescriptor(
     if (value != null & isSupportedValue(value)) {
       this.value = new StringTypeProvider(value.toString());
     } else {
-      this.value = new NothingProvider();
+      if (value != null && value.getClass() == StringTypeProvider.class
+          || value.getClass() == StringProviderImpl.class) {
+        this.value = new StringTypeProvider(((StringProviderImpl) value).getString());
```

Comment:
Checking `value instanceof StringProvider` should be equivalent while also covering other potential future sub-classes.